src/View/Helper/FlashHelper.php

```diff
@@ -26,7 +26,16 @@ class FlashHelper extends Helper
 {
 
     /**
-     * Used to render the message set in FlashComponent::set()
+     * Default config for the helper.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'stackElement' => false
```

Comment:
Oh ok. Maybe the name of the config option is misleading ?
I also thought about being able to overwrite this with the ``$options`` array of the ``render()`` method, since every other parameters can be overwritten.